lib/org/chromium/webidl/ast/Definition.js

```diff
@@ -14,6 +14,10 @@ foam.CLASS({
       of: 'org.chromium.webidl.ast.DefinitionData',
       name: 'definition',
     },
+    {
+      name: 'id',
+      factory: function() { return this.definition.id; },
```

Comment:
Not all definitions (maybe it is just `implements`) are named. Perhaps the `ids` field should be removed from `Implements` and replaced with `getName()` which returns those two objects?